lib/f_core/os/c_task.cpp

```diff
@@ -39,11 +42,11 @@ CTask::~CTask() {
 }
 
 void CTask::Initialize() {
-    for (CTenant *tenant : tenants) {
+    for (CTenant* tenant : tenants) {
```

Comment:
:| I will submit to this for the sake of consistency. It does hurt my bones tho.

Reply:
Lol